app/services/users/create_mascot_account.rb

```diff
@@ -0,0 +1,29 @@
+module Users
+  class CreateMascotAccount
+    MASCOT_PARAMS = {
+      email: "mascot@forem.com",
+      username: "mascot",
+      name: "Mascot",
+      profile_image: SiteConfig.mascot_image_url,
```

Comment:
Hmmm if this is getting created before the user has configured the site, maybe this should use one of our auto-generated images instead?

`Users::ProfileImageGenerator.call`

Reply:
We use a default image for `mascot` https://github.com/forem/forem/blob/2225fe99990fddfc529fac8017950412f63cffd2/app/models/site_config.rb#L101

Are you saying a profile image would be a better fit than the mascot image? Or that the mascot image won't work?

Reply:
Ah, gotcha. I wasn't putting it all together. Nevermind this all looks good.